packages/charts/src/interfaces/IChartBaseProps.ts

```diff
@@ -88,12 +88,22 @@ export interface IChartBaseProps<T = ICartesianChartConfig> extends Omit<CommonP
     legendPosition?: 'top' | 'bottom' | 'middle';
     /**
      * Horizontal alignment of the legend. Can be one of the following: `"left"`, `"center"`, `"right"`
+     *
+     * @deprecated Please use `legendConfig.align` instead.
      */
     legendHorizontalAlign?: 'center' | 'left' | 'right';
     /**
      * Number that sets the amount of delay time the chart waits when resizing.
      */
     resizeDebounce?: number;
+    //todo: remove "Omit" once supported
+    /**
+     * Defines the configuration object for the internally used `recharts` Legend component.
+     * You can find all possible configuration properties [here](https://recharts.org/en-US/api/Legend).
+     *
+     * __Note:__ It is possible to override internally used props, so please use with caution!
```

Comment:
would `overwrite` be more fitting here? https://stackoverflow.com/questions/8651562/overwrite-or-override#:~:text=If%20you're%20replacing%20an,or%20changing%20the%20thing%20overridden.